examples/docs_snippets/docs_snippets_tests/guides_tests/reexecution_tests/test_unreliable_pipeline.py

```diff
@@ -0,0 +1,7 @@
+from dagster import execute_pipeline
+from docs_snippets.guides.dagster.reexecution.unreliable_pipeline import unreliable_pipeline
+
+
+def test_pipeline_compiles_and_executes():
+    result = execute_pipeline(unreliable_pipeline, solid_selection=['unreliable_start', 'unreliable_end'])
```

Comment:
I don't think this solid_selection would pass. Tested it locally and got:
```
E           dagster.core.errors.DagsterInvalidConfigError: Error in config for pipeline unreliable_pipeline
E               Error 1: Missing required config entry "solids" at the root. Sample config for missing entry: {'solids': {'unreliable_end': {'inputs': {'num': '<selector>'}}}}
```
because these two solids aren't connected in the pipeline. Let's just do 'unreliable_start' for now. 

other than this, I think we are good to go!

Reply:
Ah yes. Fixed!